mne/report.py

```diff
@@ -553,7 +553,6 @@ def _build_html_slider(slices_range, slides_klass, slider_id):
     {{endif}}
     {{if comments is not None}}
         <br><br>
-        <div class="comments">
```

Comment:
...or equivalently the opening one didn't have a closing one, which makes removing the opening one a solution?